packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -19,6 +21,51 @@ const disabledInstrumentations = new Set(
 
 const loadChannel = channel('dd-trace:instrumentation:load')
 
+if (!dc.subscribe) {
```

Comment:
Why not use the API available from the polyfill (and also use the polyfill).

Reply:
This was taken mostly as-is from DCITM. If you recall from your DC refactor it's the only occasion where we still use the built-in DC instead of the polyfill. The issue is that we can't require dd-trace modules directly from within the bundled interpositional code.